js/background.js

```diff
@@ -85,3 +85,17 @@ chrome.omnibox.onInputEntered.addListener( function(text) {
         chrome.tabs.update(tabs[0].id, {url: "https://duckduckgo.com/?q="+encodeURIComponent(text)});
     });
 });
+
+//This adds Context Menu when user select some text.
+//create context menu
+chrome.contextMenus.create({
+    title : 'Search anonymously on DuckDuckGo',
+    contexts : ["selection"],
+    onclick : searchThis
+});
+
+//execute when user click on that context menu
+function searchThis(selectedText){
```

Comment:
@nirajkaushal is there any reason why this separate function needs to be created? Could it be made an anonymous function on line 94?